projects/1.18/assets/tropicraft/tropicraft/lang/zh_cn.json

```
@@ -0,0 +1,490 @@
+  "block.tropicraft.bamboo_potted_cactus": "竹盆栽仙人掌",
+  "block.tropicraft.bamboo_potted_canna": "竹盆栽美人蕉",
+  "block.tropicraft.bamboo_potted_commelina_diffusa": "竹盆栽竹节菜",
+  "block.tropicraft.bamboo_potted_cornflower": "竹盆栽矢车菊",
+  "block.tropicraft.bamboo_potted_crocosmia": "竹盆栽香鸢尾",
+  "block.tropicraft.bamboo_potted_croton": "竹盆栽巴豆",
+  "block.tropicraft.bamboo_potted_dandelion": "竹盆栽蒲公英",
+  "block.tropicraft.bamboo_potted_dark_oak_sapling": "竹盆栽深色橡木树苗",
+  "block.tropicraft.bamboo_potted_dead_bush": "竹盆栽枯木",
+  "block.tropicraft.bamboo_potted_dracaena": "竹盆栽龙血树",
+  "block.tropicraft.bamboo_potted_fern": "竹盆栽蕨",
+  "block.tropicraft.bamboo_potted_foliage": "竹盆栽树叶",
+  "block.tropicraft.bamboo_potted_grapefruit_sapling": "竹盆栽西柚树苗",
+  "block.tropicraft.bamboo_potted_iris": "竹盆栽鸢尾花",
+  "block.tropicraft.bamboo_potted_jungle_sapling": "竹盆栽丛林树苗",
+  "block.tropicraft.bamboo_potted_lemon_sapling": "竹盆栽柠檬树苗",
+  "block.tropicraft.bamboo_potted_lily_of_the_valley": "竹盆栽铃兰",
+  "block.tropicraft.bamboo_potted_lime_sapling": "竹盆栽酸橙树苗",
+  "block.tropicraft.bamboo_potted_magic_mushroom": "竹盆栽奇幻菇",
+  "block.tropicraft.bamboo_potted_mahogany_sapling": "竹盆栽桃花心木树苗",
+  "block.tropicraft.bamboo_potted_oak_sapling": "竹盆栽橡木树苗",
+  "block.tropicraft.bamboo_potted_orange_anthurium": "竹盆栽橙色火鹤花",
+  "block.tropicraft.bamboo_potted_orange_sapling": "竹盆栽橙树苗",
+  "block.tropicraft.bamboo_potted_orange_tulip": "竹盆栽橙色郁金香",
+  "block.tropicraft.bamboo_potted_orchid": "竹盆栽兰花",
+  "block.tropicraft.bamboo_potted_oxeye_daisy": "竹盆栽滨菊",
+  "block.tropicraft.bamboo_potted_palm_sapling": "竹盆栽棕榈树苗",
+  "block.tropicraft.bamboo_potted_pathos": "竹盆栽悲怆",
+  "block.tropicraft.bamboo_potted_pink_tulip": "竹盆栽粉红色郁金香",
+  "block.tropicraft.bamboo_potted_poppy": "竹盆栽罂粟",
+  "block.tropicraft.bamboo_potted_red_anthurium": "竹盆栽红色火鹤花",
+  "block.tropicraft.bamboo_potted_red_mushroom": "竹盆栽红蘑菇",
+  "block.tropicraft.bamboo_potted_red_tulip": "竹盆栽红色郁金香",
+  "block.tropicraft.bamboo_potted_spruce_sapling": "竹盆栽云杉树苗",
+  "block.tropicraft.bamboo_potted_tropical_fern": "竹盆栽热带蕨",
+  "block.tropicraft.bamboo_potted_white_tulip": "竹盆栽白色郁金香",
+  "block.tropicraft.bamboo_potted_wither_rose": "竹盆栽凋灵玫瑰",
+  "block.tropicraft.bamboo_slab": "竹台阶",
+  "block.tropicraft.bamboo_stairs": "竹楼梯",
+  "block.tropicraft.bamboo_trapdoor": "竹活板门",
+  "block.tropicraft.black_mangrove_leaves": "黑皮红树树叶",
+  "block.tropicraft.black_mangrove_log": "黑皮红树原木",
+  "block.tropicraft.black_mangrove_propagule": "红树胎生苗",
+  "block.tropicraft.black_mangrove_propagule.desc": "黑皮红树",
+  "block.tropicraft.black_mangrove_roots": "黑皮红树根茎",
+  "block.tropicraft.black_mangrove_wood": "黑皮红木",
+  "block.tropicraft.bromeliad": "菠萝植株",
+  "block.tropicraft.canna": "美人蕉",
+  "block.tropicraft.chunk": "凝灰岩",
+  "block.tropicraft.chunk_fence": "凝灰岩栅栏",
+  "block.tropicraft.chunk_fence_gate": "凝灰岩栅栏门",
+  "block.tropicraft.chunk_slab": "凝灰岩台阶",
+  "block.tropicraft.chunk_stairs": "凝灰岩楼梯",
+  "block.tropicraft.chunk_wall": "凝灰岩墙",
```

```suggestion
  "block.tropicraft.chunk": "结灰岩",
  "block.tropicraft.chunk_fence": "结灰岩栅栏",
  "block.tropicraft.chunk_fence_gate": "结灰岩栅栏门",
  "block.tropicraft.chunk_slab": "结灰岩台阶",
  "block.tropicraft.chunk_stairs": "结灰岩楼梯",
  "block.tropicraft.chunk_wall": "结灰岩墙",
```
与原版凝灰岩（Tuff）区分
本来想换成泥灰岩了事，结果发现泥灰岩也是现实存在的岩石种类，就生造了个词